sdk/template-LLC/Azure.Template.LLC/tests/Samples/TemplateServiceSamples.HelloWorld.cs

```diff
@@ -7,14 +7,14 @@
 
 namespace Azure.Template.LLC.Tests.Samples
 {
-    public class TemplateLLCSamples : SamplesBase<TemplateLLCTestEnvironment>
+    public class TemplateServiceSamples : SamplesBase<TemplateServiceTestEnvironment>
     {
         public void Authenticate()
         {
             var endpoint = TestEnvironment.Endpoint;
 
             #region Snippet:TemplateLLCAuthenticate
-            var serviceClient = new TemplateLLCClient(new DefaultAzureCredential(), new Uri(endpoint));
+            var serviceClient = new TemplateServiceClient(new DefaultAzureCredential(), new Uri(endpoint));
```

Comment:
Would it make sense to Create Get and Delete in the sample?

Reply:
Added samples [here](https://github.com/Azure/azure-sdk-for-net/blob/b5027f09f31f9de9c5fe911dde7ce461f71f93ed/sdk/template-LLC/Azure.Template.LLC/tests/Samples/TemplateServiceSamples.HelloWorld.cs). We run samples as a part of tests, so I commented out [[Test]](https://github.com/Azure/azure-sdk-for-net/blob/b5027f09f31f9de9c5fe911dde7ce461f71f93ed/sdk/template-LLC/Azure.Template.LLC/tests/Samples/TemplateServiceSamples.HelloWorld.cs#L27). It will give an idea that we need to enable the tests when writing SDK for the service.